README.md

```diff
@@ -346,7 +346,52 @@ sudo systemclt enable --now sentrum.service
 
 6. Check the logs with `journalctl -fu sentrum`
 
-# Future Work
+## Docker
+
+To run Sentrum using Docker, you can either build the image yourself or use the prebuilt image.
```

Comment:
nit: I prefer to keep referring to `sentrum` in lowercase, for consistency.

Reply:
Fixed in https://github.com/sommerfelddev/sentrum/pull/4/commits/e3494ecb7eccbde6f5969818f4539a92328c5a00